src/config/Config.zig

```diff
@@ -126,6 +126,8 @@ pub const compatibility = std.StaticStringMap(
 ///
 /// Changing this configuration at runtime will only affect new terminals, i.e.
 /// new windows, tabs, etc.
+///
+/// Available since: 1.0.0
```

Comment:
I think we can remove the 1.0.0 ones and have that be implied. That'll help clean up a lot of the noise in the docs and here.

Reply:
Done.